### PR TITLE
z3 Release

### PR DESCRIPTION
**Pull request by ebfull**
For the `z3` release.

### CODE REVIEW

**Comment by str4d** [comment]
Are these changes following any pattern, or are they just random?

**Comment by ebfull** [comment]
I'm just choosing arbitrarily. We can come up with a more rigorous magic later.

**Comment by str4d** [comment]
Then ACK.

**Comment by ebfull** [comment]
**FORCE PUSHED**

**Comment by str4d** [comment]
Reviewing!

File: src/clientversion.h [hunk 1]
```diff
@@ -16,7 +16,7 @@
 //! These need to be macros, as clientversion.cpp's and bitcoin*-res.rc's voodoo requires it
 #define CLIENT_VERSION_MAJOR 0
 #define CLIENT_VERSION_MINOR 11
-#define CLIENT_VERSION_REVISION 2
+#define CLIENT_VERSION_REVISION 3
```
Comment (nathan-at-least) [inline src/clientversion.h]: NACK. This is the minor revision number ``N`` in ``v0.11.<N>.z3``. I don't think the ``.z3`` is embodied anywhere in this file, so let's remove mention of this file in ``release-process.md``.

**Comment by str4d** [comment]
ACK!

File: src/clientversion.h (hunk 1 above)
Comment (str4d) [inline src/clientversion.h]: Glad that someone else picked up on this!

**Comment by ebfull** [comment]
Force pushed again.

**Comment by str4d** [comment]
ACK!

**Comment by nathan-at-least** [comment]
ACK.

@zkbot r+

**Pull request merged**